bin/benchmark

```diff
@@ -7,6 +7,7 @@ $VERBOSE = nil
 
 require "bundler/setup"
 require "sorbet-runtime"
+require "tmpdir"
```

Comment:
I'm confused why this is failing in CI, since this succeeds:

```
ruby -e "require 'tmpdir'; puts Dir.tmpdir"
```

Reply:
Ah, it's because that's from the `main` branch:
https://github.com/Shopify/ruby-lsp/blob/main/.github/workflows/benchmark.yml#L15

So this should be fine to merge.